Doc/tutorial/inputoutput.rst

```diff
@@ -329,11 +329,30 @@ equivalent :keyword:`try`\ -\ :keyword:`finally` blocks::
 
 If you're not using the :keyword:`with` keyword, then you should call
 ``f.close()`` to close the file and immediately free up any system
-resources used by it. If you don't explicitly close a file, Python's
-garbage collector will eventually destroy the object and close the
-open file for you, but the file may stay open for a while.  Another
-risk is that different Python implementations will do this clean-up at
-different times.
+resources used by it. :meth:`__del__` of the returned :term:`file object` also calls ``f.close()``.
```

Comment:
```suggestion
resources used by it.
```

This topic is very early in the tutorial. `__del__` method shouldn't be mentioned.

Reply:
But where else should we mention it? In the documentation of :term:`file object` ?

Reply:
https://docs.python.org/3/library/io.html#io.IOBase.__del__

Reply:
Ok